src/filesystem.cpp

```diff
@@ -247,7 +247,24 @@ namespace fc {
   void copy( const path& f, const path& t ) {
      boost::system::error_code ec;
      try {
+        #if (BOOST_VERSION/100000) == 1 && ((BOOST_VERSION/100)%1000) > 73
```

Comment:
This check may not work as expected if boost ever goes to 2.0. I believe all other checks for boost version in eosio & fc use something like
```c++
#if BOOST_VERSION >= 107100
```
IMO that's more readable too. If you have time, may want to consider this instead.

Reply:
sure, it is good to make this change. thanks!